roles/internal/activemq/tasks/install.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Clean activemq path
+  file:
+    state: absent
+    path: "{{ activemq_install_root }}/activemq"
+
+- name: Download and unarchive activemq
+  unarchive:
+    remote_src: yes
+    src: http://archive.apache.org/dist/activemq/{{ activemq_version }}/apache-activemq-{{ activemq_version }}-bin.tar.gz
+    dest: "{{ activemq_install_root }}"
+    owner: "{{ activemq_user }}"
+    group: "{{ activemq_group }}"
+
+- name: Move activemq
+  command: mv "{{ activemq_install_root }}/apache-activemq-{{ activemq_version }}" "{{ activemq_install_root }}/activemq"
+
+- name: Create symlink
+  file:
+    state: link
+    src: "/opt/activemq/bin/activemq"
+    dest: "/etc/init.d/activemq"
+
+- name: Add activemq service to start up
+  shell: 'update-rc.d activemq defaults'
```

Comment:
This also needs to be made idempotent somehow. I would suggest dropping this entirely and handling it all using the ansible services module in the task below.